test/postgresql/import_test.rb

```diff
@@ -1,4 +1,7 @@
 require File.expand_path(File.dirname(__FILE__) + '/../test_helper')
 require File.expand_path(File.dirname(__FILE__) + '/../support/postgresql/import_examples')
 
+pg_version = ENV["PG_VERSION"] || "0"
```

Comment:
Reminder to self that I should probably create a test readme file so options like this can be surfaced. 

Out of curiosity what did you use to determine to how to set up and run the test environment? Just figure it out by looking through code or was there any information that was helpful any where in the repos or on the internets?